<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchFellowDlg</class>
 <widget class="QDialog" name="SearchFellowDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找好友</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>8</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="searchEdit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="resultListWidget">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>取消</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>打开会话</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
